core/invehicle-digital-twin/src/main.rs

```diff
@@ -96,7 +95,22 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         info!("This service is not using Chariott.");
     }
 
-    server_future.await?;
+    // sample_layer can be set to None if you wish to disable it
```

Comment:
if the sample layer is going to stay as part of the core service, then consider allowing this via config rather than needing a code change

Reply:
The intent is not to let this remain here for very long.  It is a temporary placeholder.  Devin will soon replace it when he adds his Agemo-Ibeji extension, which will replace it as an example and he will use a feature flag to decide whether it gets used or not.